src/qt/forms/optionsdialog.ui

```diff
@@ -163,6 +163,57 @@
             </property>
            </widget>
           </item>
+          <item>
+           <widget class="QGroupBox" name="groupBoxAddressType">
```

Comment:
This is inside Export group box, should be on the outside instead?

Reply:
Fixing.